spec/game_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'game'
+
+describe Game do
+  
+  let(:game) { Game.new(1) }  
+
+  context 'I want the game to have 10 frames' do
+    
+    it 'ends' do
+      expect{ game.start_game }.to output(include("Game Over")).to_stdout
+      end
+    end
+
+    it 'runs 10 times' do
+      expect(game).to receive(:start_frame).exactly(10).times
+      game.start_game 
+    end
+  end
+           
```

Comment:
Good start on getting the testing going. This is testing behaviour over state which is what the aim of the project was.